packages/botonic-plugin-flow-builder/src/webview/use-webview-contents.ts

```diff
@@ -19,16 +19,43 @@ export function useWebviewContents({
   botId,
   webviewId,
   locale,
+  mapContents,
 }: UseWebviewContentsProps): UseWebviewContents {
   const [textContents, setTextContents] = useState<WebviewTextContent[]>()
   const [imageContents, setImageContents] = useState<WebviewImageContent[]>()
+  const [contents, setContents] = useState({})
   const [currentLocale, setCurrentLocale] = useState(locale)
-  const [isLoading, setLoading] = useState(false)
+  const [isLoading, setLoading] = useState(true)
   const [error, setError] = useState(false)
 
+  const getTextContent = (contentID: string): string => {
+    return (
+      textContents
+        ?.find(textContent => textContent.code === contentID)
+        ?.content.text.find(text => text.locale === currentLocale)?.message ||
+      ''
+    )
+  }
+
+  const getImageSrc = (contentID: string): string => {
+    return (
+      imageContents
+        ?.find(imageContent => imageContent.code === contentID)
+        ?.content.image.find(image => image.locale === currentLocale)?.file ||
+      ''
+    )
+  }
+
+  const createContentsObject = () => {
+    const contentsObject = {}
+    for (const [key, value] of Object.entries(mapContents)) {
+      contentsObject[key] = getTextContent(value)
+    }
```

Comment:
Change: You're missing the image sources to be obtanined also.
```suggestion
    for (const [key, contentId] of Object.entries(mapContents)) {
     
      contentsObject[key] = imageContents?.find(imageContent => imageContent.code === contentId) ? getImageSrc(contentId) :  getTextContent(contentId)
    }
```